pyxrf/xanes_maps/xanes_maps_api.py

```diff
@@ -443,6 +391,124 @@ def build_xanes_map_api(start_id=None, end_id=None, *, param_file_name,
         logger.info("Processing is complete.")
 
 
+def _load_data_from_databroker(*, start_id, end_id, wd_xrf):
+    """
+    Implements the first step of processing sequence: loading the batch of scan data
+    from databroker.
+
+    Parameters
+    ----------
+
+    start_id : int
+        first scan ID of the batch of scans
+
+    end_id : int
+        last scan ID of the batch of scans
+
+    wd_xrf : str
+        full (absolute) name of the directory, where loaded .h5 files are placed
+    """
+
+    # Try to create the directory (does nothing if the directory exists)
+    os.makedirs(wd_xrf, exist_ok=True)
+    files_h5 = [fl.path for fl in os.scandir(path=wd_xrf) if fl.name.lower().endswith(".h5")]
+    if files_h5:
+        logger.warning(f"The temporary directory '{wd_xrf}' is not empty. "
+                       f"Deleting {len(files_h5)} files (.h5) ...")
+        for fln in files_h5:
+            logger.info(f"Removing raw xrf data file: '{fln}'.")
+            os.remove(path=fln)
+    make_hdf(start_id, end_id, wd=wd_xrf,
+             completed_scans_only=True, file_overwrite_existing=True,
+             create_each_det=False, save_scaler=True)
+
+
+def _process_xrf_data(*, start_id, end_id, wd_xrf, param_file_name, emission_line,
+                      incident_energy_low_bound, use_incident_energy_from_param_file):
+    """
+    Implements the second step of the processing sequence: processing of XRF scans
+    and generation of XRF maps
+
+    Parameters
+    ----------
+
+    start_id : int
+        first scan ID of the batch of scans
+
+    end_id : int
+        last scan ID of the batch of scans
+
+    wd_xrf : str
+        full (absolute) name of the directory, where loaded .h5 files are placed
+
+    param_file_name : str
+        the name of the JSON parameter file. The parameters are used for automated
+        processing of data with ``pyxrf_batch``. The parameter file is typically produced
+        by PyXRF.
+
+    emission_line : str
+        the name of the selected emission line ("Ca_K", "Fe_K", etc.). The emission line
+        of interest.
+
+    incident_energy_low_bound : float
+        files in the set are processed using the value of incident energy which equal to
```

Comment:
remove "which" from "...value of incident energy which equal to..."

Reply:
Fixed